### ISSUE TITLE
idn_to_ascii(): INTL_IDNA_VARIANT_2003 is deprecated on PHP 7.2+

### ISSUE THREAD

**Issue by jaygilmore** [freescout-helpdesk/freescout#1301]
We're seeing errors and blocking of views on PHP 7.2+ on version 1.7.11. This is not new. It seems to be that emails that come in that contain images are causing this issue. We reverted to PHP 7.1 which is not ideal but then the Modules stopped working. It may be caused by HTMLPurifier but this [appears to have been fixed](https://github.com/ezyang/htmlpurifier/issues/163). I'm not sure what can be done about it? I assume with the autoloader, it should be using recent releases. 

Web Server: nginx/1.16.1
PHP: PHP 7.3.11
OS: centOS6

The following is the error we're seeing without the trace: 
```
idn_to_ascii(): INTL_IDNA_VARIANT_2003 is deprecated (View: www/resources/views/conversations/partials/thread.blade.php) (View: www/resources/views/conversations/partials/thread.blade.php) (View: www/resources/views/conversations/partials/thread.blade.php) {"userId":1,"email":"jay@modx.com","exception":"[object] (ErrorException(code: 0): idn_to_ascii(): INTL_IDNA_VARIANT_2003 is deprecated (View: www/resources/views/conversations/partials/thread.blade.php) (View: www/resources/views/conversations/partials/thread.blade.php) (View: www/resources/views/conversations/partials/thread.blade.php) at www/vendor/ezyang/htmlpurifier/library/HTMLPurifier/AttrDef/URI/Host.php:103, ErrorException(code: 0): idn_to_ascii(): INTL_IDNA_VARIANT_2003 is deprecated (View: www/resources/views/conversations/partials/thread.blade.php) (View: www/resources/views/conversations/partials/thread.blade.php) at www/vendor/ezyang/htmlpurifier/library/HTMLPurifier/AttrDef/URI/Host.php:103, ErrorException(code: 0): idn_to_ascii(): INTL_IDNA_VARIANT_2003 is deprecated (View: www/resources/views/conversations/partials/thread.blade.php) at www/vendor/ezyang/htmlpurifier/library/HTMLPurifier/AttrDef/URI/Host.php:103, ErrorException(code: 0): idn_to_ascii(): INTL_IDNA_VARIANT_2003 is deprecated at www/vendor/ezyang/htmlpurifier/library/HTMLPurifier/AttrDef/URI/Host.php:103)
```

The following is what we see when visiting a message that contains an image: 
![image](https://user-images.githubusercontent.com/378199/124003771-b0942300-d9ad-11eb-9f41-0466cf094bbf.png)

**Comment by freescout-helpdesk** [comment]
`INTL_IDNA_VARIANT_2003 is deprecated` - this is deprecation notice. It can't cause error 500. Check logs in Manage > Logs > App Logs.

**Comment by freescout-helpdesk** [comment]
Closing for no activity.

**Comment by jaygilmore** [comment]
I was away last week. I agree a deprecation notice will not cause a 500 but a deprecation notice can also happen while something else up the chain fails silently causing the 500. The application fails and reports the error with no logged 500 when there are images in the emails. This is handled by HTMLPurifier and this was a known issue but "fixed" in 4.12. It's possible there were residual issues and that it may be resolved in 4.13.0 which was released June 28, 2020. It may not at all be related to HTMLpurifier but we've not been able to determine any other culprit that handles the parsing of the email messages.

**Comment by freescout-helpdesk** [comment]
https://github.com/freescout-helpdesk/freescout/commit/b8f3200f8fef299a79144143d7444e6125391f27
